src/dlgProfilePreferences.cpp

```diff
@@ -2080,6 +2080,12 @@ void dlgProfilePreferences::saveMap()
     QTimer::singleShot(10 * 1000, this, &dlgProfilePreferences::hideActionLabel);
 }
 
+QString dlgProfilePreferences::mapSaveLoadDirectory(Host* pHost) {
```

Comment:
Maybe just `mapsDirectory`?

Reply:
My initial thought was exactly like that... but it can return profile directory as well :)

Reply:
I don't think the name reflects that it returns the profile dir, and it's OK not to be super precise here in an not often used helper function